### ISSUE TITLE
Fails with error when installed on RHEL8

### ISSUE THREAD

**Issue by thopri** [cinatic/taskwhisperer#91]
Have tried installing this extension on RHEL8 which has gnome 3.32 installed which this extension is valid for?

Get red error box, using looking glass (Alt +F2, type lg and then select extensions, show errors). 

Error is graphene not found, typelib file for namespace 'graphene' not found.

A google is not suggesting anything obvious so was wondering if the was a known issue and what the fix could be?

**Comment by cinatic** [comment]
thanks for the report, i got it running on gnome 3.32 but it looked a bit ugly, as it looks perfect on above version it might be an issue with my test ubuntu vm. Could you verify if it looks good for your and if everything is working as expected:

https://github.com/cinatic/taskwhisperer/pull/new/gnome-3.32-compat
[taskwhisperer-extension.zip](https://github.com/cinatic/taskwhisperer/files/6461289/taskwhisperer-extension.zip)

**Comment by thopri** [comment]
yes that works great on RHEL8 although as you say a little ugly but all the functionality seems to be there. Thanks for getting that working as the extension now works across all my devices....
